DOCUMENTATION.md

```diff
@@ -89,7 +89,7 @@ If a template has placeholder fields for personalised information such as name o
 A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. If you do not have a reference, you must pass in an empty string or `null`. For example:
 
 ```
-"your_reference_here";
+"your_reference_here"
```

Comment:
Should this be using single quotes too?

Reply:
Yeah likely, I think since this is not using ```javascript standard-markdown doesnt lint it?